### PR TITLE
Adding a note for content staging preview

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Adding a note that only users with admin permissions can see shared preview

This pull request (PR) ...

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/cms/content-staging-preview.html

- ...
- ...

### CODE REVIEW

**Comment by dobooth** [comment]
Hi @DmitryTsymbal Please look into the linting errors here.

**Comment by DmitryTsymbal** [comment]
Hi @dobooth , thank you for noticing. I have fixed all errors here

File: src/cms/content-staging-preview.md [hunk 1]
```diff
@@ -52,4 +52,7 @@ _Preview page controls_
 
    - Paste the link into the body of an email message.
 
+     {:.bs-callout-info}
+     Only Admin users are allowed to see the preview, you are sharing. It is required to create admin account for user before the sharing, if it does not exist.Otherwise, user won’t be able to see a preview.
```
Comment (leslietilling) [inline src/cms/content-staging-preview.md:56]:
```suggestion
     An Admin user account is required to see a shared preview. If you have access to [create a new Admin user account]({% link system/permissions-users-all.md %}), you must create the account for a new user before the sharing.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**